common/jenkins-agents/python/docker/Dockerfile.ubi8

```diff
@@ -13,10 +13,11 @@ ENV PYTHONUNBUFFERED=1 \
 RUN yum module install -y python38:3.8/build && \
     yum module install -y python39:3.9/build --allowerasing && \
     yum install -y python3.11 python3.11-pip python3.11-devel python3.11-setuptools --allowerasing && \
+    yum install -y python3.12 python3.12-pip python3.12-devel python3.12-setuptools --allowerasing && \
```

Comment:
As 3.8 reaches EOL, drop 3.8?

Reply:
since the agent is still also supporting 3.6 I would not drop 3.8 for now, also before dropping any of those legacy ones we should ensure we are not affecting anyone, I will double check with the team.
nevertheless, I see this happening in another issue/PR.

Reply:
@braisvq1996  FYI